app/javascript/packs/application.ts

```diff
@@ -1,6 +1,7 @@
 require('sweetalert/dist/sweetalert.css');
 
 import start from 'retrospring/common';
+import initAnnouncements from 'retrospring/features/announcement/index';
```

Comment:
```suggestion
import initAnnouncements from 'retrospring/features/announcement';
```

You don't need to add `index` to a folder import if there is an `index.{js.ts}` file present.